deployment_and_ssl.md

```diff
@@ -0,0 +1,72 @@
+# Deploying Healthlocker to an Azure instance
+
+Most of the instruction will be found in the
+[Oxleas deployment documentation](https://github.com/healthlocker/oxleas-adhd/blob/master/deployment_and_ssl_doc.md#create-an-ssl-certificate).
+
+This document will go through changes in the deployment process that need to
+happen in order to deploy Healthlocker.
+
+### [Installing Erlang](https://github.com/healthlocker/oxleas-adhd/blob/master/deployment_and_ssl_doc.md#install-erlang)
+
+Erlang needs to be installed with ODBC to access the MSSQL database.
+`apt-get install erlang-odbc`
+
+### Install ODBC drivers
+After installing all necessary languages and dependencies, ODBC drivers also
+need to be installed.
+```
+curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
+curl https://packages.microsoft.com/config/ubuntu/16.04/prod.list > /etc/apt/sources.list.d/mssql-release.list
+apt-get update
+ACCEPT_EULA=Y apt-get install msodbcsql -y
+```
+
+```
+# optional: for bcp and sqlcmd
+ACCEPT_EULA=Y apt-get install mssql-tools -y
+# Add tools to path
+echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bash_profile
+echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bashrc
+source ~/.bashrc
+# optional: for unixODBC development headers
+apt-get install unixodbc-dev -y
+```
+
+### [Adding Environment Variables](https://github.com/healthlocker/oxleas-adhd/blob/master/deployment_and_ssl_doc.md#add-environment-variables-to-azure)
+
+In addition to all the environment variables listed in the Oxleas
+documentation, the following variables also need to be added to the server.
+
+```
+SEGMENT_WRITE_KEY
+READ_ONLY_HOSTNAME
+READ_ONLY_USERNAME
+READ_ONLY_DATABASE
+READ_ONLY_PASSWORD
+MPass
+```
+
+### Setting up roles
+You first need to create the database and tables.
```

Comment:
Lines 51-56 are setup commands and straight copies from the Oxleas document?  Should this document specify that those stages be followed from the Oxleas doc?  Line 57, `sudo -u postgres psql` should only be done once the (as yet not-admin) admin user has been created within the app online.

Reply:
@reddog It's mentioned below that you can only update once it has been added on the live site. Would you prefer that postgres is not even connected to until the user has been made?

Reply:
@katbow You are right.  I think my comment was meant to be in two parts - lines 51-56, should they not be just references to the Oxleas doc?  Maybe just reverse the order of the commentary below and the postgres connection just so that the reader only connects to postgres once the user has been created?  Just wanting to help ensure that a database connection isn't opened, the read goes off and does something else whilst the admin user is being created, and then *forgets* to come back and close it.

Reply:
okay gotcha. In this case I decided not to refer to the oxleas docs since there were other commands in this section on oxleas. I thought this was the most clear option for which set of commands were necessary to run for Healthlocker, despite some repetition. 

I will switch things around though so that it is:
* Add user on website with email you want to use for admin
* Open postgres connection on the server
* Update the role for the entered email to "admin"